plugins/kapt3/kapt3-compiler/src/org/jetbrains/kotlin/kapt3/Kapt3Plugin.kt

```diff
@@ -229,7 +229,7 @@ class Kapt3ComponentRegistrar : ComponentRegistrar {
         try {
             Class.forName(JAVAC_CONTEXT_CLASS)
         } catch (e: ClassNotFoundException) {
-            logger.warn("'$JAVAC_CONTEXT_CLASS' class can't be found ('tools.jar' is absent in the plugin classpath). Kapt won't work.")
+            logger.error("'$JAVAC_CONTEXT_CLASS' class can't be found ('tools.jar' is absent in the plugin classpath). Kapt won't work.")
```

Comment:
Maybe it's better to use strong warning here? See `CompilerMessageSeverity.STRONG_WARNING`

Reply:
Well, it's the other possible solution, but annotation processing phase will be skipped then, and the user likely get an error such as "Unresolved reference <...>" later.
So I think this is closer to error.